src/main/java/it/trade/api/StatelessTradeItApiClient.java

```diff
@@ -311,4 +311,57 @@ public void onSuccessResponse(Response<TradeItGetAllTransactionsHistoryResponse>
             }
         });
     }
+
+    public void previewCryptoOrder(
+            TradeItPreviewCryptoOrderRequest request,
```

Comment:
Should this be `final` too?

Reply:
Actually the final is not necessary here even for the callback.
I added it when first implementing the java api because of the anonymous class, but as we pass the callback in the constructor, it is not necessary anymore. Will clean it up.

Reply:
We've been using `let` over `var` in Swift and I think it's helped our code to be cleaner. I'd vote for using `final` more consistently. Up to you if you have the time to fix up your latest commit though.